conans/test/unittests/util/apple_test.py

```diff
@@ -44,22 +37,28 @@ def test_to_apple_arch(self):
         self.assertIsNone(tools.to_apple_arch('mips'))
 
     def test_apple_sdk_name(self):
-        self.assertEqual(tools.apple_sdk_name(FakeSettings('Macos', 'x86')), 'macosx')
```

Comment:
If the intent (it should be) is not to break existing calls, leave these tests unmodified, and exclusively add new tests.

Reply:
okay, sure, I'll change